lib/twine/formatters.rb

```diff
@@ -1,30 +1,13 @@
-require 'twine/formatters/abstract'
-require 'twine/formatters/android'
-require 'twine/formatters/apple'
-require 'twine/formatters/flash'
-require 'twine/formatters/gettext'
-require 'twine/formatters/jquery'
-require 'twine/formatters/django'
-require 'twine/formatters/tizen'
-
 module Twine
   module Formatters
-    @formatters = [Formatters::Apple, Formatters::Android, Formatters::Gettext, Formatters::JQuery, Formatters::Flash, Formatters::Django, Formatters::Tizen]
+    @formatters = []
 
     class << self
       attr_reader :formatters
-
-      ###
-      # registers a new formatter
-      #
-      # formatter_class - the class of the formatter to register
-      #
-      # returns array of active formatters
-      #
-      def register_formatter formatter_class
-        raise "#{formatter_class} already registered" if @formatters.include? formatter_class
-        @formatters << formatter_class
-      end
```

Comment:
Speaking of all of this, I wonder if it would be a good idea to create a document on the new formatters with instructions on how to write one and how to implement it as a plugin. Not necessary for this PR, but it might be good to start thinking about it, especially for the next major version of Twine (and especially as it gets closer to 1.0).